js/friday.js

```diff
@@ -1,34 +1,38 @@
 (function IsItFridayYet() {
   const date = new Date();
-  const theMonth = date.getMonth();
-  const theDay = date.getDay();
-  const element = document.getElementById("friday");
-  const spans = document.getElementsByClassName("fallingElement");
+  const thisMonth = date.getMonth();
+  const thisDay = date.getDay();
+  const theBody = document.body;
+  const textElement = document.getElementById("friday");
   let season = "";
 
-  if (theDay === 5) {
-    element.innerHTML = "YES";
+  // return a programatic yes on Fridays to print on homepage.
+  function isFriday(n) {
+    return (n == 5 ? "YES" : "NO");
+  }
 
```

Comment:
Using a Switch statement is genius, however it is not as popular of a solution in modern-day JS. The reason is that they are prone to errors, and does not allow for modern ES6 syntax (to name a few issues). As such, we will instead use the tried and true if/else conditions below.